appengine/standard_python3/bundled-services/deferred/flask/main.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, request
+from google.appengine.api import wrap_wsgi_app
+from google.appengine.ext import deferred
+from google.appengine.ext import ndb
+
+my_key = 'main'
+
+app = Flask(__name__)
+app.wsgi_app = wrap_wsgi_app(app.wsgi_app, use_deferred=True)
+
+
+class Counter(ndb.Model):
+    count = ndb.IntegerProperty(indexed=False)
+
+
+def do_something_later(key, amount):
+    entity = Counter.get_or_insert(key, count=0)
+    entity.count += amount
+    entity.put()
+
+
+@app.route('/counter/increment')
+def increment_counter():
+    # Use default URL and queue name, no task name, execute ASAP.
+    deferred.defer(do_something_later, my_key, 20)
+
+    # Use default URL and queue name, no task name, execute after 60s.
+    deferred.defer(do_something_later, my_key, 20, _countdown=60)
+
+    # Providing non-default task queue arguments
+    deferred.defer(
+        do_something_later, my_key, 20, _url='/custom/path', _countdown=60)
+
+    return 'Deferred counter increment.'
+
+
+@app.route('/counter/get')
+def view_counter():
+    counter = Counter.get_or_insert(my_key, count=0)
+    return str(counter.count)
+
+
+@app.route('/custom/path', methods=['POST'])
```

Comment:
I assume `/custom/path` is the correct value, and not a placeholder?

Reply:
It is. I could be almost anything, actually. It's the address for an internal POST for deferred handling. The app sets the path and the handler both to /custom/path.

Note that this is not intended to serve external requests at this path.